documentation-website/src/components/cookie-consent.tsx

```diff
@@ -0,0 +1,55 @@
+import React, {
+  useState,
+} from 'react';
+import './cookie-consent.css';
+import {
+  Lang,
+} from './lang';
+
+const CookieConsent = () => {
+  const [
+    consentWasClosed,
+    setConsentWasClosed,
+  ] = useState<boolean>(false);
+
+  function handleConsent(accept: boolean) {
+    { /*@ts-expect-error _paq can be null*/ }
```

Comment:
why not use `?.`?

Reply:
without comment  '?.' still gives an error